Radio/index.jsx

```diff
@@ -157,7 +158,7 @@ const Radio = React.createClass({
                     </div>
                   </div>}
 
-                  <div className={classNames(classes.optionRight)}>
+                  <div className={aside ? classNames(classes.optionRight) : classNames(classes.optionRight, classes.optionRightmost)}>
```

Comment:
It’s probably better to write this using the "object" notation for the `classNames` function:

```javascript
className={classNames(
  classes.optionRight, 
  {
    [classes.optionRightmost]: aside != null
  }
)}
```

Not a big deal though